doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -679,22 +679,47 @@ Now fetch additional sources from the Eclipse OpenJ9 project and its clone of Ec
 bash get_source.sh
 ```
 
-### 3. Create the Docker image
+### 3. Prepare for build on AArch64 Linux
 
-Run the following commands to build a Docker image for AArch64 cross-compilation, called **openj9aarch64**:
+You must install a number of software dependencies to create a suitable build environment on your AArch64 Linux system:
+
+- GNU C/C++ compiler
```

Comment:
Do you want to be more descriptive about which version?  e.g., 7.3

Reply:
Added GCC version to this line.